src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -8991,9 +8994,6 @@ static bool TakeOutParam(object y, out object x)
             var compilation = CreateCompilationWithMscorlib45(source, references: new[] { ValueTupleRef, SystemRuntimeFacadeRef },
                                                               options: TestOptions.DebugExe, parseOptions: TestOptions.Regular);
             compilation.VerifyDiagnostics(
-                // (11,13): error CS1023: Embedded statement cannot be a declaration or labeled statement
```

Comment:
If the LDM decides to reverse course and change the specification back to a declaration, we can add the error back.